app/models/account.rb

```diff
@@ -15,7 +16,9 @@ class Account < ApplicationRecord
   has_many :notifications, dependent: :destroy
 
   validates :uuid, presence: true, uniqueness: true
-  validates :idme_uuid, presence: true, uniqueness: true
+  validates :idme_uuid, uniqueness: true
```

Comment:
I'm wondering if we need to relax this validation (and the corresponding database unique constraint if it exists).

Reply:
what use cases are you thinking that would cause a collision on idme_uuid?

Reply:
I'm not sure, just that we've seen almost every kind of data anomaly we can think of. I know we've seen the same user with two ID.me wallets, but that would be different ID.me UUIDs and hence not a collision. I guess the converse would be a person who winds up with two different SECIDs that map to the same ID.me UUID. I can't quite imagine the scenario that would lead to that. I guess for now leaving the constraint in place should be fine, we'd probably see a unique constraint violation get raised and visible in Sentry if it occurred.